Get offline batch inference details using task API in m (#8305)
opensearch-project/documentation-website: _ml-commons-plugin/api/model-apis/batch-predict.md

## Prerequisites

> Before using the Batch Predict API, you need to create a connector to the externally hosted model.  For every action, specify the  `action_type` parameter that describes the action:

Reviewer: There are extra spaces before "For every" and after "specify the". I would say "For each action".

## Prerequisites

> Before using the Batch Predict API, you need to create a connector to the externally hosted model. For example, to create a connector to an OpenAI `text-embedding-ada-002` model, send the following request. The optional `action_type` paramter supports canceling the batch job running on OpenAI:

Reviewer 2: The optional action_type parameter allows for checking the status or canceling a batch job running on OpenAI.

Author: Can we have something like this: 
Before using the Batch Predict API, you need to create a connector to the externally hosted model. For example, to create a connector to an OpenAI `text-embedding-ada-002` model, send the following request. The action_type `cancel_batch_predict` is optional and it supports canceling the batch job running on OpenAI:

> Before using the Batch Predict API, you need to create a connector to the externally hosted model. For example, to create a connector to an OpenAI `text-embedding-ada-002` model, send the following request. The optional `action_type` paramter supports canceling the batch job running on OpenAI:

Reviewer 3: ```suggestion
Before using the Batch Predict API, you need to create a connector to the externally hosted model. For example, to create a connector to an OpenAI `text-embedding-ada-002` model, send the following request. The optional `action_type` parameter supports canceling the batch job running on OpenAI:
```

> Before using the Batch Predict API, you need to create a connector to the externally hosted model. For example, to create a connector to an OpenAI `text-embedding-ada-002` model, send the following request. The optional `action_type` paramter supports cancelling the batch job that is running on OpenAI:

Reviewer: ```suggestion
Before using the Batch Predict API, you need to create a connector to the externally hosted model. For example, to create a connector to an OpenAI `text-embedding-ada-002` model, send the following request. The optional `action_type` paramter supports canceling the batch job running on OpenAI:
```

#### Example response

The response contains the task ID for the batch predict operation:

```json
{
  "task_id": "KYZSv5EBqL2d0mFvs80C",
  "status": "CREATED"
}
```

> To check the status of the batch job operation, provide the task ID to the [Tasks API]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api/tasks-apis/get-task/). You can find the job details in the `remote_job` field in the task. Once the prediction is complete, the task `state` changes to `COMPLETED`.

Reviewer: "batch job" => "batch predict"?

Reviewer 3: Reworded.

> To check the status of the batch job operation, provide the task ID to the [Tasks API]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api/tasks-apis/get-task/). You can find the job details in the `remote_job` field in the task. Once the prediction is complete, the task `state` changes to `COMPLETED`.

Reviewer 3: ```suggestion
To check the status of the batch predict job, provide the task ID to the [Tasks API]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api/tasks-apis/get-task/). You can find the job details in the `remote_job` field in the task. Once the prediction is complete, the task `state` changes to `COMPLETED`.
```
